### ISSUE TITLE
[Bug Report] Newly set up bot doesn't join a server

### ISSUE THREAD

**Issue by SleeplessSloth** [jagrosh/MusicBot#363]
**Describe the bug**
After setting up the bot for the first time, it doesn't do anything after loading up, i.e. neither does it join the server I invited it in or react to commands

**To Reproduce**
1. Start the bot with `java -Dnogui=true -jar JMusicBot-0.2.7-Linux.jar `

**Expected behavior**
The bot should join the server and react to commands

**Actual behavior**
The channel greeted it but the bot is nowhere to be seen. It doesn't appear in the member list.

**Debug Info**

> Please run the `debug` command and paste it here or upload it as an attachment. If you cannot run the command, please provide information about your system such as operating system version, JMusicBot version, etc.

I'm not sure how to do that. I've tried adding `debug` to the end of the start command but the output was the same, i.e. `java -Dnogui=true -jar JMusicBot-0.2.7-Linux.jar debug`

**Log**
```
[21:45:27] [WARN] [Settings]: Failed to load server settings (this is normal if no settings have been set yet): java.nio.file.NoSuchFileException: serversettings.json
[21:45:28] [INFO] [Startup]: Loaded config from /home/gray/bot/config.txt
[21:45:30] [INFO] [JDA]: Login Successful!
[21:45:31] [INFO] [WebSocketClient]: Connected to WebSocket
[21:45:31] [INFO] [JDA]: Finished Loading!
```

**Additional info**
In the beginning I was running the bot in a Debian container with openjdk JRE 11 but that didn't work. Then I've tried running it on the host (Raspi 3B+) with Arch Linux ARM and openjdk JRE 13 and on my main PC with the regular Arch Linux and openjdk JRE 13 - same result. So I'm guessing it's either a configuration issue or even a discord one. In the config I've only changed the token, owner id, and the prefix. I've also rechecked and even regenerated the token a couple of times and re-invited it to the server but to no avail.

**Comment by jagrosh** [comment]
This sounds like you're either not running the same bot as you think you are, or you have your server's permissions set up incorrectly. Either way, there's nothing I can do to assist you here, you'll need to investigate those possibilities (a bot joining a server has no relation to the code that you run).

**Comment by jagrosh** [comment]
Also the `debug` command is an in-discord bot command (like `ping` or `about`).

**Comment by SleeplessSloth** [comment]
Hmm, seems like you're right. I've temporary set the @everyone role to be an admin (on my private server, not to worry) and the bot connected almost immediately. Then I've changed its role to the role I specifically created for bots and I've reverted the @everyone role, and now it seems to connect just fine. Super strange... I do have a very limited @everyone role, i.e. it's not allowed to message or use voice chat. But usually people just appear in the members list and I move them to their appropriate role manually, including other music bots like Rhythm Bot. But JMusicBot didn't appear in the members list at all, strange...
